### PR TITLE
Add sso-rbac module for gke cluster

### PR DESCRIPTION
**Pull request by VahagnMian**

### CODE REVIEW

File: modules/sso-rbac/provider.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+provider "kubernetes" {
```
Comment (mrdntgrn) [inline modules/sso-rbac/provider.tf:1]: please remove provider blocks and related data blocks from module, instead have them inside repo where you are going to use this module

File: modules/sso-rbac/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!-- BEGIN_TF_DOCS -->
```
Comment (mrdntgrn) [inline modules/sso-rbac/README.md:1]: have some minimal description of module here

Comment (VahagnMian) [reply modules/sso-rbac/README.md:1]: Actually there is an example directory, and in that directory there are two files describing module usage

Comment (mrdntgrn) [reply modules/sso-rbac/README.md:1]: this is not about example, just have some description what this module is for, where how it works,
no muck words just basic description for one who open it and want to understand what it is for

File: modules/sso-rbac/variables.tf [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#variable "cluster_name" {
+#  type = string
+#}
+#
+#variable "cluster_location" {
```
Comment (mrdntgrn) [inline modules/sso-rbac/variables.tf:5]: it seems you can remove this variable from here and from examples

File: modules/sso-rbac/README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
+This is a module which will configure `GKE RBAC` to sync with Google groups
+
+```
+module "gcp-sso-module" {
```
Comment (mrdntgrn) [inline modules/sso-rbac/README.md:4]: please set module source from terraform registry, and also have the version line,

remove all following locals, data, provider blocks. just left one this module, put minimal role binding inside the module scopes,

remove cluster_location and cluster_name variable here and 2 examples you have as this module have no this variables anymore

**Review by mrdntgrn — APPROVED** [review]

**Pull request merged**